src/python/chrome/build_info.py

```diff
@@ -22,6 +23,7 @@
                       '[^,]*,[^,]*,[^,]*,[^,]*,[^,]*,'
                       '([0-9a-f]+),.*')
 BUILD_INFO_URL = 'https://omahaproxy.appspot.com/all?csv=1'
+BUILD_INFO_URL_CD = 'https://chromiumdash.appspot.com/fetch_releases?num=1&platform=%s'  # pylint: disable=line-too-long
```

Comment:
Can we use a named format for this. i.e. "platform={platform}". Then later `str.format(platform=...)`

Can split up this line by doing

```python
BUILD_INFO_URL_CD = (
   'https://chromiumdash.appspot.com/fetch_releases?'
   'num=1&platform={platform}')
```